spec/remote_fetching_spec.rb

```diff
@@ -111,9 +103,19 @@
   end
 
   it 'sends provided HTTP headers in the request' do
+    # Faraday is required only after calling .parse_http
+    # This line is just to trigger this require, then it's possible to
+    # add an expectation of how Faraday is initialized after.
+    FormatParser.parse_http('invalid_url') rescue nil
+
+    expect(Faraday)
```

Comment:
If you `expect('Faraday')...`, does it still complain about the missing constant?

Reply:
it doesn't work! 😞 

<img width="781" alt="image" src="https://user-images.githubusercontent.com/302303/108365019-046c4f80-71f7-11eb-901f-a71ca087cfd0.png">

Reply:
Bummer. Then, whatever you see fit. The last approach looks very hacky, I think the previous one looked more purposeful, even though it was more involved.

Reply:
I prefer the current one because it ensures that all headers were sent.

It's indeed a bit hacky, but I think it's easy to see what is going on reading the comment that I added

ok?